dipy/denoise/noise_estimate.py

```diff
@@ -42,7 +45,7 @@ def piesno(data, N, alpha=0.01, l=100, itermax=100, eps=1e-5,
     alpha : float
         Probabilistic estimation threshold for the gamma function.
 
-    l : int
+    ell : int
```

Comment:
Since you are updating this can you pick a more self-explanatory variable name ?

Reply:
Done in e635078.